yb-voyager/src/srcdb/yugabytedb.go

```diff
@@ -362,12 +445,83 @@ func (yb *YugabyteDB) FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sql
 	return nonEmptyTableList, emptyTableList
 }
 
-func (yb *YugabyteDB) GetTableColumns(tableName *sqlname.SourceName) ([]string, []string, []string) {
-	return nil, nil, nil
+func (yb *YugabyteDB) GetTableColumns(tableName *sqlname.SourceName) ([]string, []string, []string, error) {
+	var columns, dataTypes, dataTypesOwner []string
+	query := fmt.Sprintf(GET_TABLE_COLUMNS_QUERY_TEMPLATE_PG_AND_YB, tableName.ObjectName.Unquoted, tableName.SchemaName.Unquoted)
+	rows, err := yb.conn.Query(context.Background(), query)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("error in querying(%q) source database for table columns: %w", query, err)
+	}
+	defer rows.Close()
+	for rows.Next() {
+		var column, dataType, dataTypeOwner string
+		err = rows.Scan(&column, &dataType, &dataTypeOwner)
+		if err != nil {
+			return nil, nil, nil, fmt.Errorf("error in scanning query(%q) rows for table columns: %w", query, err)
+		}
+		columns = append(columns, column)
+		dataTypes = append(dataTypes, dataType)
+		dataTypesOwner = append(dataTypesOwner, dataTypeOwner)
+	}
+	return columns, dataTypes, dataTypesOwner, nil
 }
 
-func (yb *YugabyteDB) GetColumnsWithSupportedTypes(tableList []*sqlname.SourceName, useDebezium bool, _ bool) (map[*sqlname.SourceName][]string, []string) {
-	return nil, nil
+func (yb *YugabyteDB) getDatatypesThatAreUserDefinedTypesOtherThanEnumOrDomain(columns, dataTypes []string) []string {
```

Comment:
I guess this func is better called `filterUnsupportedUserDefinedDatatypes` because tomorrow this list of enum or domain might change.

do add a comment mentioning that as of now, all UDTs except enum and domain are unsupported (which reflects in your query)